corehq/apps/geospatial/templates/case_grouping_map.html

```diff
@@ -53,6 +53,12 @@
       to apply the polygon filtering changes.
     {% endblocktrans %}
   </div>
+  <div class="alert alert-warning" data-bind="visible: hasUrlError">
+    {% blocktrans %}
+      There are too many polygons on the map. Please remove some before refreshing the page, otherwise the
+      newly added polygons will be ignored when filtering.
```

Comment:
That is correct. If the new polygon causes the URL length to exceed max then it won't be added to the URL, however we'll still keep track of the new polygon in `self.polygons`. This means that if a user deletes a polygon then the newly added polygon should get added to the URL. In saying that, I wonder if we should keep track of the new polygon or just ignore it completely?

Reply:
Hey @zandre-eng 

I am a bit confused by that.

My original feedback was just regarding the message shown to the user. I was thinking the message should be updated to
"otherwise the new added polygons will be lost".

About the cases you mentioned, I am unable to put that together.

Reply:
> My original feedback was just regarding the message shown to the user.

Ah. apologies for the misunderstanding, I thought you wanted to know more about how we handle the logic for when there are too many polygons. The suggested wording sounds good to me.

> About the cases you mentioned, I am unable to put that together.

Please do let me know if you'd like me to clear up any confusion regarding my previous comment and how we handle too many polygons in the code.

Reply:
[ca9610b](https://github.com/dimagi/commcare-hq/pull/34511/commits/ca9610b06d15bb13768ddfcecdac63ca7ff51a30)